documentation/tools.md

```diff
@@ -19,63 +19,88 @@ An example Cube is specified in [cube.ttl](cube.ttl). The cube provides a constr
 ### Validate the cube
```

Comment:
I would actually propose making this a lvl 2 header and raising subsections with it. I see not reason to nest under `Example Cube`

Reply:
💯